exploration/number-selection.md

```diff
@@ -116,20 +181,52 @@ but can cause problems in target locales that the original developer is not cons
 > [*] {{You have {$var} chances remaining}}
 > ```
 
-Additional options such as `minimumFractionDigits` and others already supported by `:number`
-should also be supported.
+### Other Numeric Selector Functions
+
+The function `:plural` operates identically to `:number`, except:
+- the `option` of `select` is not valid
+- the `select` type of this selector is always `plural`
+- the function is not valid as a formatter
+
+The function `:ordinal` operates identically to `:number`, except:
+- the `option` of `select` is not valid
+- the `select` type of this selector is always `ordinal`
+  (This function is valid as a formatter)
+
+The function `:integer` operates identically to `:number`, except:
+- the `minimumFractionDigits` and `maximumFranctionDigits` options
+  default to `0` even for floating point numeric types
+> [!NOTE]
+> The fraction digits can be overridden for the purposes of formatting
+> and selection, but the result must be that any fractional part of
+> the operand is reduced to `0`
```

Comment:
What does this mean?

Reply:
It means that this:

```
{|1.2345| :integer minimumFractionDigits=1}
```

Formats as:

```
1.0
```

Reply:
That seems rather surprising. If we do that, it'll be the only place in the built-in functions where we're transforming the inputs. Do we really need this? Couldn't we just not support `minimumFractionDigits` on `:integer`?

Reply:
We don't really transform the inputs, but I see what you mean. I'll forbid all of the fraction digits on `:integer` for now.